docs/reference/content/bson/pojos.md

```diff
@@ -389,7 +393,7 @@ public final class Person {
 The `Person` POJO Will produce BSON similar to:
 
 ```json
-{ "_id": "1234567890", "_t": "Person", "firstName": "Alan", "surname": "Turing",
+{ "_id": "ObjectId(111111111111111111111111)", "_t": "Person", "firstName": "Alan", "surname": "Turing",
```

Comment:
Wouldn't it just store the hex?

Reply:
I think it amounts to the same time, this is just using shell syntax.  But better to use standard extended JSON syntax in the docs, IMO.

Reply:
I changed to extended JSON with "$oid" as per this: https://docs.mongodb.com/manual/reference/mongodb-extended-json/#bson.ObjectId